### ISSUE TITLE
Referenced assemblies not being loaded

### ISSUE THREAD

**Issue by ivankahl** [TheBerkin/Docpal#2]
When I try and generate documentation for a library that references other DLLs that have been copied to the output folder as well when building a project, it does not load the other assemblies located in the same folder:

Getting the following error:

```text
Unfortunately, there was an error.

System.IO.FileNotFoundException: Could not load file or assembly 'ProcessBase, Version=1.0.3.1, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
File name: 'ProcessBase, Version=1.0.3.1, Culture=neutral, PublicKeyToken=null'
   at System.Reflection.RuntimeAssembly.GetExportedTypes(RuntimeAssembly assembly, ObjectHandleOnStack retTypes)
   at System.Reflection.RuntimeAssembly.GetExportedTypes()
   at Docpal.DocpalPages.BuildDocs(String outputPath) in D:\Personal Projects\Programming\Visual C#\docpal-github\Docpal\DocpalPages.cs:line 44
   at Docpal.Program.Main(String[] args) in D:\Personal Projects\Programming\Visual C#\docpal-github\Docpal\Program.cs:line 66
```

I changed the `Assembly.LoadFile` to `Assembly.LoadFrom` on Line 36 and it resolves the issue.

**Comment by ivankahl** [comment]
I have created a pull request and fixed the bug here #3

**Comment by TheBerkin** [comment]
Fixed in #3
